shell/probe/raindrop.ps1

```diff
@@ -52,11 +52,12 @@ while ($true) {
             exit 1
         } else {
             $Dat = ""
-            Remove-Item $Dir -Force -Recurse
+            Remove-Item $Dir -Force -Recurse -ErrorAction SilentlyContinue
```

Comment:
what if we put this logic into the catch block and raise an exception here in the else block? I like that because it's duplicate logic that we'll want (it'd be nice to drop the $Dat in the catch as well). 

this also gives us a nice print statement after each test loop.

Reply:
Love it